clusterdock/topologies/cdh/actions.py

```diff
@@ -49,8 +65,15 @@ def start(args):
         args.registry_url, args.namespace or DEFAULT_CLOUDERA_NAMESPACE,
         args.cdh_string, args.cm_string
     )
-
     images = [primary_node_image, secondary_node_image]
+
+    if args.kerberos:
+        logger.info('Adding KDC node image ...')
```

Comment:
Probably don't need to log this; can display info about KDC stuff when the container is being handled.